test/index.test.ts

```diff
@@ -76,7 +76,7 @@ test('load index.(x)', async () => {
   expect(output[0].code).toMatchInlineSnapshot(`
     "class Foo {
       render() {
-        return React.createElement(\\"div\\", {
+        return /* @__PURE__ */ React.createElement(\\"div\\", {
```

Comment:
I think this is just from an esbuild version bump. Not sure how to disable this?